src/main/host/memory_manager/mod.rs

```diff
@@ -431,6 +431,19 @@ impl MemoryManager {
         Ok(unsafe { res.assume_init() })
     }
 
+    /// Writes a local value `val` into the memory at `ptr`.
+    ///
+    /// ```ignore
```

Comment:
Maybe just `no_run` instead of `ignore`? https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#attributes

Reply:
Added (and updated example in `read`).